src/states/node.rs

```diff
@@ -61,8 +62,8 @@ use xor_name::XorName;
 const TICK_TIMEOUT_SECS: u64 = 60;
 /// Time (in seconds) after which a `GetNodeName` request is resent.
 const GET_NODE_NAME_TIMEOUT_SECS: u64 = 60;
-/// Time (in seconds) the members of a section wait for a joining node they sent a network name to.
-const SENT_NETWORK_NAME_TIMEOUT_SECS: u64 = 30;
+/// The number of required leading zero bits for the resource proof
+const RESOURCE_PROOF_DIFFICULTY: u8 = 4;
```

Comment:
The new node will only need to try on average 16 hash attempts to satisfy this - shouldn't it be somewhat harder?